plotMDTS_function.R

```diff
@@ -131,18 +168,51 @@ plotMDTS <- function(
   }
   
   # Lines to plot
-  if (!is.null(group)) {
+  if (!is.null(group) && !is.null(time.shift)) {
+    
+    # Adding group and time.shift
```

Comment:
Just define the additional `geom_line`s as variables and add them, e.g., I think you should be able to do:

```
n1 = geom_line(...)
n2 = geom_line(...)
n3 = geom_line(...)

switch(time.shift, data.plot + n1, data.plot + n1 + n2, data.plot + n1 + n2 + n3)
```

I guess you could even do:

```
ns = (n1, n2, n3)
for (i in 1:time.shift) {
  data.plot = data.plot + ns[i]
}
```